.docker/entrypoints/cli/03-provision-site.sh

```
@@ -20,21 +20,7 @@ set -e
-
-# Delegate Drupal provisioning to the Quant-aware script. The standard
-# DrevOps provision script is not compatible with Quant Cloud because it relies
-# on Lagoon-specific tooling (e.g., rsync to Lagoon SSH).
-if [ -x "./scripts/quant/provision-quant.sh" ]; then
-  ./scripts/quant/provision-quant.sh
-else
-  echo "Quant provisioning script missing or not executable." >&2
-  exit 1
-fi
+ if [ "${DREVOPS_PROVISION_USE_PROFILE}" = "1" ]; then
+   export DREVOPS_PROVISION_OVERRIDE_DB="$(cat .env | grep ^DREVOPS_PROVISION_OVERRIDE_DB | cut -c31-)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)